Sources/LLVM/PointerType.swift

```diff
@@ -24,6 +24,12 @@ public struct PointerType: IRType {
   /// - parameter addressSpace: The optional address space where the pointed-to
   ///   object resides.
   public init(pointee: IRType, addressSpace: Int = 0) {
+    // FIXME: This class of invalid reference is not caught by Module.verify(),
+    // only `lli`.
+    if LLVMGetTypeKind(pointee.asLLVM()) == LLVMVoidTypeKind {
```

Comment:
`pointee is VoidType`, no? `LLVMTypeRef` doesn't conform to `IRType` and it's guaranteed that an `IRType` with kind `LLVMVoidTypeKind` is going to be `VoidType`